.github/workflows/deployment.yml

```diff
@@ -0,0 +1,28 @@
+name: Deployment
+
+on:
+  # Uncomment once we are ready to deploy 
+  #push:
+  #  branches:
+  #    - main
+  workflow_dispatch:
+
+jobs:
+  deploy-web:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout workspace
+      uses: actions/checkout@v4
+    - name: Update docker image
```

Comment:
Hey @SPDonaghy, I want to get this PR merged in because it's blocking another PR I opened up: https://github.com/UBCSailbot/sailbot_workspace/pull/386

If everything else looks good to you, would you mind approving this PR? I created a new issue for this task https://github.com/UBCSailbot/sailbot_workspace/issues/387, so @JordanChen123 can work on it separately.

Reply:
Yeah of course!